tests/objc-cli/libmanaged/Tests/Tests.m

```diff
@@ -90,10 +90,6 @@ - (void)testConstructors {
 
 	id super_unique_default_init = [[Constructors_SuperUnique alloc] init];
 	XCTAssert ([super_unique_default_init id] == 411, "super id");
-
-	// FIXME - this should not be allowed, that .ctor is not available to call in .NET as it is not re-declared in SuperUnique
```

Comment:
Created an issue so we do not forget https://github.com/mono/Embeddinator-4000/issues/104